invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_login.html

```diff
@@ -6,68 +6,70 @@
+                <i class="fitted dropdown icon inverted"></i>
+                <div class="menu">
+                    {%- for item in plus_menu_items if item.visible %}
+                        <a class="item"
+                           style="color:black;"
+                           href="{{ item.url }}">{{ item.text|safe }}
+                        </a>
+                    {%- endfor %}
+                </div>
+            </div>
+        {% endif %}
 
-    {%- if config.USERPROFILES %}
-      <div class="ui buttons user-profile">
-        <a class="ui button"
-           href="{{ url_for('invenio_userprofiles.profile') }}">
-          <i class="user icon"></i> {{ current_user.email|truncate(31,true) }}
-        </a>
+        {%- if config.USERPROFILES %}
+            <div class="ui buttons user-profile">
+                <a class="ui button"
+                   href="{{ url_for('invenio_userprofiles.profile') }}">
+                    <i class="user icon"></i> {{ current_user.email|truncate(31,true) }}
+                </a>
 
-        <div class="ui dropdown icon button floating">
-          <i class="dropdown icon"></i>
-          <div class="menu">
-            {%- for item in current_menu.submenu('settings').children if item.visible %}
-              <a class="item"
-                style="color:black;"
-                href="{{ item.url }}">{{ item.text|safe }}
-              </a>
+                <div class="ui dropdown icon button floating">
+                    <i class="dropdown icon"></i>
+                    <div class="menu">
+                        {%- for item in current_menu.submenu('settings').children if item.visible %}
+                            <a class="item"
+                               style="color:black;"
+                               href="{{ item.url }}">{{ item.text|safe }}
+                            </a>
 
-            {%- endfor %}
-            <div class="divider"></div>
-            <a class="item" style="color:black;" href="{{ url_for_security('logout') }}">
-              <i class="sign-out icon"></i>
-              {{ _('Log out') }}
-            </a>
-          </div>
-        </div>
-      </div>
+                        {%- endfor %}
+                        <div class="divider"></div>
+                        <a class="item" style="color:black;" href="{{ url_for_security('logout') }}">
+                            <i class="sign-out icon"></i>
+                            {{ _('Log out') }}
+                        </a>
+                    </div>
+                </div>
+            </div>
 
-    {%- else %}
-      <a href=" {{ url_for_security('logout') }}" class="ui button"><i
-              class="sign-out icon"></i> {{ _('Log out') }}</a>
+        {%- else %}
+            <a href=" {{ url_for_security('logout') }}" class="ui button"><i
+                    class="sign-out icon"></i> {{ _('Log out') }}</a>
+        {%- endif %}
     {%- endif %}
-  {%- endif %}
-{%- endif %}
+{%- endif %}
```

Comment:
I think you should add a new line at the end of the file.

Reply:
will do :+1: